### DISCUSSION TITLE
Profile ==> notifications ==> "Server Error (500)"

### DISCUSSION THREAD

**Discussion by pmatuszy** [healthchecks/healthchecks#756] [General]
Hi
a question about notification:
I have a version 2.4.1 and when I try to go to "email reports" I get "Server Error (500)"
I think this never worked as I didn't try to configure it. In errorlog of my gunicorn there is no error so I am not sure where I should look for problems. Have you got any ideas where to start my troubleshooting from?
Regards
Paul

**Comment by cuu508** [comment]
> Have you got any ideas where to start my troubleshooting from?

Run Healthchecks with DEBUG=False, you should then see the error details in the HTTP 500 response page.

**Reply by pmatuszy** [reply]
Ok - done it now
```
ValueError at /accounts/profile/notifications/
ZoneInfo keys must be normalized relative paths, got: Europe//Paris
Request Method:	GET
Request URL:	https://192.168.1.249:7900/accounts/profile/notifications/
Django Version:	4.1.2
Exception Type:	ValueError
Exception Value:	
ZoneInfo keys must be normalized relative paths, got: Europe//Paris
Exception Location:	/usr/lib/python3.10/zoneinfo/_tzpath.py, line 91, in _validate_tzfile_path
Raised during:	hc.accounts.views.notifications
Python Executable:	/encrypted/webapps/hc-venv/bin/python
Python Version:	3.10.6
Python Path:	
['/encrypted/webapps/healthchecks',
 '/encrypted/webapps/hc-venv/bin',
 '/usr/lib/python310.zip',
 '/usr/lib/python3.10',
 '/usr/lib/python3.10/lib-dynload',
 '/encrypted/webapps/hc-venv/lib/python3.10/site-packages']
Server time:	Wed, 21 Dec 2022 08:32:51 +0000
```
I've added pytz to my virtual env, restarted gunicorn but it didn't help. Have you got any idea how to progress from that point?

**Reply by cuu508** [reply]
I'm not sure how it got there, but looks like the zone name `Europe//Paris` has one extra slash. Can you try the following:

* on your Healthchecks instance, open `/admin/accounts/profile/`
* look up your account's profile in the list, open its change form
* in the change form, what's in the "Tz" field? 
* If the value there has double forward slashes, delete one, and save changes.

**Reply by pmatuszy** [reply]
hmmm - i can't edit - i can "login" and "delete" profiles.... Am I missing opions?

**Reply by cuu508** [reply]
To navigate to the "change" form, click on the numeric ID next to the profile.

**Reply by pmatuszy** [reply]
yes - double '/' was the problem. I wonder how it got there as I never edited this. 

PROBLEM IS NOW SOLVED. BIG BIG RESPECT. THANK YOU!
